src/components/SQForm/SQFormCheckbox.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {Checkbox, CheckboxProps} from '@material-ui/core';
```

Comment:
So CheckboxProps is a type that we'd like to use `import type` on. However, we can't do `import type {Checkbox, CheckboxProps} from '@material-ui/core';` since `Checkbox` isn't a type and we need it at run time.

So we'll need two import statements from the same the module. One for type and one for the component like below.

```
import {Checkbox} from '@material-ui/core';
import type {CheckboxProps} from '@material-ui/core';
```

ESlint might complain at you for duplicate types. If it does, I found this github issue with the solution https://github.com/eslint/eslint/issues/13669#issuecomment-690085156

Would you be able to comb all the files again and split up type and non-types imports? And if ESLint complains then include the above solution in your PR as well?

Reply:
@20BBrown14 Will do this.  Thanks for the feedback.